src/layout/section/landingGridSection/LandingGridSection.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import { HeroSection, HeroSectionProps } from '../heroSection'
+import { LandingGridSectionElements } from './LandingGridSection.style'
+
+export type LandingGridSectionProps = HeroSectionProps &
+  Readonly<{
+    children: React.ReactNode
+  }>
+
+export const LandingGridSection = ({
```

Comment:
I would drop the *Grid* from the naming (folder and file name), the fact that it uses a grid internally is an implementation detail.
Other way to phrase it: if we start using grids for other components or section, do we need to add grid in their name ?

Reply:
The main thing that this "component does is to set a "Grid Layout" for this page. Should I call it **Layout** then?

Reply:
`SEALandingLayout` maybe? The fact that this use a Grid Layout could change in the future. It's like a design implementation detail.

Reply:
Resolved here: [Renaming](https://github.com/blablacar/ui-library/pull/1065/commits/69e4198ccf19034c72563e2762b99240d8fa1c93)